### PR TITLE
feat: feeds

### PR DESCRIPTION
**Pull request by zen87-byte**
## Story/Task
https://trello.com/c/DPwo3u3k

## Summary
- Create required attach url
- delete svg file and change into react-icons

## Screenshot
1. Null
Screenshot 2023-08-06 at 23 44 42

2. Add New Post
Screenshot 2023-08-08 at 22 53 39
Screenshot 2023-08-08 at 22 53 50
Screenshot 2023-08-08 at 22 53 58

3. Uploaded Feeds List
Screenshot 2023-08-06 at 23 49 57
Screenshot 2023-08-06 at 23 50 05

4. Edit Feed
Screenshot 2023-08-08 at 23 28 15
Screenshot 2023-08-08 at 23 29 16

5. Delete Feed
Screenshot 2023-08-06 at 23 50 30

### CODE REVIEW

**Review by msyahrulsp — CHANGES_REQUESTED** [review]
- Si button attach image atau attach url ilangin aja. Soalnya input url itu required. Simpennya di atas input text article aja. labelnya `Attachment URL`
- `feeds.svg` hapus aja. Ambil komponen `Header` dari staging
- Hapus folder `img`, si isinya pindah `images`
- Buat icon, pake `react-icons` aja

File: src/pages/feeds.tsx [hunk 1]
```diff
@@ -0,0 +1,428 @@
...
+    });
+
+
+    const { isOpen: isPostOpen, onOpen: onPostOpen, onClose: onPostClose } = useDisclosure();
+    const { isOpen: isRemoveOpen, onOpen: onRemoveOpen, onClose: onRemoveClose } = useDisclosure();
+    const { isOpen: isEditOpen, onOpen: onEditOpen, onClose: onEditClose } = useDisclosure();
+    const [editId, setEditId] = useState<string | null>(null);
+    const [editedArticle, setEditedArticle] = useState('');
+    const [editedUrl, setEditedUrl] = useState('');
+    console.log('Edited feed: ', editedArticle);
```
Comment (YakobusIP) [inline src/pages/feeds.tsx:72]: tolong dihapus ya

File: src/pages/feeds.tsx [hunk 2]
```diff
@@ -0,0 +1,428 @@
...
+                                                variant='unstyled'
+                                                border={formState.errors.url ? '1px solid red' : '1px solid black'}
+                                                px='0.5rem'
+                                                py='0.25rem'
+                                                width='100%'
+                                                bgColor='white'
+                                                focusBorderColor='none'
+                                                borderRadius='8px'
+                                                height='400px'
+                                                {...register('article', {
```
Comment (YakobusIP) [inline src/pages/feeds.tsx:225]: bahasanya jangan article deh kayanya, ubah ke content aja

**Review by YakobusIP — CHANGES_REQUESTED** [review]

File: src/pages/feeds.tsx (hunk 1 above)
Comment (zen87-byte) [reply src/pages/feeds.tsx]: untuk rhfnya nanti aku tambahin lagi, saat ini aku hapus dulu local statenya kak

**Pull request merged**